src/components/TeamProjectCard.vue

```diff
@@ -43,7 +43,7 @@ export default Vue.extend({
   width: 320px;
   background-position: center;
   background-repeat: no-repeat;
-  background-size: 100%;
+  background-size: cover;
```

Comment:
We might have a conflict with this since I changed the css a bit, but I can quickly fix https://github.com/ubclaunchpad/new/pull/45/commits/6ca7ba68fc29bc793467f1d1b537e741bd6c1c48
I can change

Reply:
Updated: https://github.com/ubclaunchpad/new/pull/45/commits/ea7d27d75c2b7cc8f807493b400d0f333efbf61c

Reply:
Thanks for taking care of that on your end!